docs/man/git-lfs-migrate.1.ronn

```diff
@@ -108,9 +111,67 @@ The following configuration:
 
 Would, therefore, include commits: F, E, D, C, B, but exclude commit A.
 
-The presence of flag --everything indicates that all local references should be
+The presence of flag `--everything` indicates that all local references should be
 migrated.
 
+## EXAMPLES
+
+### Migrate unpushed commits
+
+The migrate command's most common use case is to convert large git objects to
+LFS before pushing your commits. By default, it only scans commits that don't
+exist on any remote.
+
+First, run `git lfs migrate info` to list the file types taking up the most
+space in your repository.
+
+```
+$ git lfs migrate info
+migrate: Fetching remote refs: ..., done
+migrate: Sorting commits: ..., done
+migrate: Examining commits: 100% (1/1), done
+*.mp3  	284 MB	  1/1 files(s)	100%
+*.pdf  	42 MB 	  8/8 files(s)	100%
+*.psd  	9.8 MB	15/15 files(s)	100%
+*.ipynb	6.9 MB	  6/6 files(s)	100%
+*.csv  	5.8 MB	  2/2 files(s)	100%
+```
+
+Now, you can run `git lfs migrate import` to convert some file types to LFS:
+
+```
+$ git lfs migrate import --include="*.mp3" --include="*.psd"
+migrate: Fetching remote refs: ..., done
+migrate: Sorting commits: ..., done
+migrate: Rewriting commits: 100% (1/1), done
+  mp3	d2b959babd099fe70da1c1512e2475e8a24de163 -> 136e706bf1ae79643915c134e17a6c933fd53c61
```

Comment:
Is `mp3` a local ref? This may be confusing as `mp3` is an include pattern given to this command. Can we change this to `master`?

Reply:
Ah yup!